README.md

```diff
@@ -35,21 +32,22 @@ git clone https://github.com/BIG-MAP/BattINFO.git
 
 When opening BattINFO.ttl in Protégé, the correct version of emmo-inferred will be downloaded and imported.
 
-In EMMO-python correct import is obtained with:
+In EMMO-python, correct import is obtained with:
 
 ```python
 from emmo import get_ontology
+
 # Loading from local repository
 ontobatt = get_ontology('/path/to/BattINFO.ttl').load(url_from_catalog=True)
+
 # Loading from web (not yet available due to closed repo)
-# ontobatt = get_ontology('https://raw.githubusercontent.com/BIG-MAP/BattINFO/master/BattINFO.ttl').load()
+ontobatt = get_ontology('https://raw.githubusercontent.com/BIG-MAP/BattINFO/master/BattINFO.ttl').load()
 ```
 
-
-Attributions and credits
-------------------------
+## Attributions and credits
 
 ### Contributors
+
 - Francesca Lønstad Bleken, SINTEF, Norway
 - Jesper Friis, SINTEF, Norway
 - Simon Clark, SINTEF, Norway
```

Comment:
I think Simon should be listed first

Reply:
Sure. I'll change that. I didn't change the order in this PR until now though, just added an empty new line. But I think it makes sense for Simon to be first on this.